scripts/solve_network_together.py

```diff
@@ -260,6 +260,9 @@ def add_entsoe_demand(n):
     )
     n.stores.loc[store_i, "capital_cost"] = store_cost 
 
+    if not snakemake.config["scenario"]["h2_pipelines"]:
```

Comment:
I would suggest combining this with setting the storage cost to medium pressure tank in the lines before or at least making the hydrogen storage costs in the background system another option in the config as well.
So e.g. having the switch in the config `hydrogen_backbone: True/False` and if `True` we have cheap underground storage + pipelines or we add another switch only for the storage like `h2_underground_store: True/False`

Reply:
Good idea. I can implement this tomorrow